App.js

```diff
@@ -1,61 +1,83 @@
 import { NavigationContainer } from '@react-navigation/native';
```

Comment:
The standard convention is to use only lowercase for repo names and branch names. So this repo would be "ciao-frontend-mobile" instead of "Ciao-Frontend-Mobile".  More info here: https://digilent.com/reference/software/development/git/start#:~:text=Repository%20Names,only%20lower%20case%20alphanumeric%20characters.